server-backend/src/SwapInRunner.ts

```diff
@@ -70,16 +70,31 @@ export class SwapInRunner {
         }
     }
 
+    private async retrySendPayment(invoice: string, cltvLimit: number, retries: number = 3, initialDelay: number = 300000, backoffFactor: number = 2): Promise<Buffer> {
```

Comment:
```suggestion
    private async retrySendPayment(invoice: string, cltvLimit: number, retries = 3, initialDelay = 300000, backoffFactor = 2): Promise<Buffer> {
```
The types are implicit and it reads a bit better.